README.md

```diff
@@ -77,3 +81,184 @@ You should see activity in the ngrok logs, server logs, and output to the vista.
 - if BMC has anything S3-compat, could use that as well to avoid Minio
 
 Other ideas: Feel free to create GitHub issues or contact Chad
+
+## Understanding How This Works
+
+In this sort of mashup, the most important thing to understand are the
+boxes and the lines. The boxes are the services and the lines are the
+wiring between the services. Let's look at the wiring first.
+
+### Wiring
+
+The wiring for the demo uses a few enablers: ngrok and minio.
+
+ngrok is an insanely useful tool that essentially opens up a big
+security hole in your home router for as long as ngrok runs. This is
+why the demo cannot run on a corporate guest WiFi. This is deemed an
```

Comment:
actually I don't think we know whether ngrok will work in a particular corporate net/guest net ... Can you just say "May not work on Corp networks and guest wifi's"?